src/pattern-library/components/Library.tsx

```diff
@@ -48,7 +49,10 @@ export type LibraryPageProps = {
 function Page({ children, intro, title }: LibraryPageProps) {
   return (
     <section className="max-w-6xl pb-16 space-y-8 text-slate-7">
-      <div className="sticky top-0 z-4 h-16 flex items-center bg-slate-0 border-b">
+      <div
+        className="sticky top-0 z-4 h-16 flex items-center bg-slate-0 border-b"
+        id="page-header"
```

Comment:
This is the page's sticky header. Just giving it an ID so I can query for it in the `PlaygroundApp`'s navigation/scroll handling `useEffect` hook.